.coveragerc

```diff
@@ -1,24 +1,59 @@
 [run]
+concurrency = multiprocessing
 parallel = True
+sigterm = True
+branch = True
 
 [report]
-include = pythonrepo*,tests*
+include = pak/*,tests/*
 # Regexes for lines to exclude from consideration
 exclude_lines =
     # Have to re-enable the standard pragma
     pragma: no cover
     except Exception
     except BaseException:
     # Don't complain if tests don't hit defensive assertion code:
+    except UnicodeDecodeError
     raise AssertionError
     raise NotImplementedError
     raise ImportError
-    except unittest.SkipTest
-    except IOError
-    except OSError
-
+    raise baton
+    baton = ImportError
+    raise ModuleNotFoundError
+    baton.module
```

Comment:
🙅🏻 These exclude specific debug code patterns. The current implementation is acceptable because the use-case of the pattern is specific enough to use such specific exclusion patterns.